web/src/views/DandisetLandingView/DownloadDialog.vue

```diff
@@ -122,7 +122,7 @@ import CopyText from '@/components/CopyText.vue';
 
 function formatDownloadCommand(identifier: string, version: string): string {
   if (version === 'draft') {
-    return `dandi download https://dandiarchive.org/dandiset/${identifier}/draft`;
+    return `dandi download ${window.location.origin}/dandiset/${identifier}/draft`;
```

Comment:
@aaronkanzer do you think you would have time to add custom handling below for `DANDI:` as well? it is pretty much "use DANDI: if window.location.origin == 'https://dandiarchive.org'", and otherwise needs a `${window.location.origin}/dandiset/` prefix

Reply:
Updated here -- let me know if this is what you had in mind -- thanks again

Reply:
I think it looks good. 

Should/could we proceed here or any other aspect remains open?